Sync/Synchronizers/Bookmarks/BookmarksSynchronizer.swift

```diff
@@ -118,34 +118,37 @@ open class BufferingBookmarksSynchronizer: TimestampedSingleCollectionSynchroniz
 
         statsSession.start()
         if !AppConstants.shouldMergeBookmarks {
-            if case .release = AppConstants.BuildChannel {
-                // On release, just mirror; don't validate.
-                run = doMirror
-            } else {
-                run = doMirror >>== effect({ result in
-                    // Just validate to report statistics.
-                    if case .completed = result {
-                        log.debug("Validating completed buffer download.")
-                        let _ = buffer.validate()
+            run = doMirror >>== { result in
```

Comment:
I'm okay with it. I would recommend timing it and including the timings in the ping; flying blind and not knowing how long we are spending on every sync would be a risk.